airflow/dags/kernel_changes.py

```diff
@@ -376,14 +376,45 @@ def IssueFactory(data, journal_id, issue_order):
     issue.year = metadata.get("publication_year")
     issue.volume = metadata.get("volume", "")
     issue.number = metadata.get("number", "")
-    issue.label = metadata.get(
-        "label", "%s%s" % ("v" + issue.volume, "n" + issue.number)
-    )
     issue.order = metadata.get("order", 0)
     issue.pid = metadata.get("pid", "")
     issue.journal = models.Journal.objects.get(_id=journal_id)
     issue.order = issue_order
 
+    def _get_issue_label(metadata: dict) -> str:
+        """Produz o label esperado pelo OPAC de acordo com as regras aplicadas
+        pelo OPAC Proc e Xylose.
+        
+        Args:
+            metadata (dict): conteúdo de um bundle
+        
+        Returns:
+            str: label produzido a partir de um bundle
+        """
+
+        label_number = metadata.get("number", "")
+        label_volume = metadata.get("volume", "")
+        label_supplement = (
+            " suppl %s" % metadata.get("supplement", "")
+            if metadata.get("supplement", "")
```

Comment:
Se o valor de suplemento é 0, como fica o resultado?

Se o valor for 0, deve apresentar apenas Suppl. Detalhe. A palavra "suppl" deve ser internacionalizada.

Reply:
Olha @robertatakenaka o 0 estava ficando na string como o esperado, no OPAC estava acontecendo algum tratamento e o 0 era removido quando necessário (pois é). Agora eu copiei a implementação via regex do xylose, acho que agora não teremos pontos obscuros 🎉

Reply:
🚀